.github/workflows/release.yml

```
@@ -0,0 +1,57 @@
+name: Release 3.x
+
+on:
+  push:
+    tags:
+      - v3.*
+  workflow_dispatch:
+
+jobs:
+  release-3.x:
```

actionlint (1.7.4)

invalid job ID "release-3.x". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)